cmd/ssh-sign/main.go

```diff
@@ -188,37 +187,36 @@ func main() {
 		}
 
 		// Parse the public key into SSH wire format
-		pak, _, _, _, err := ssh.ParseAuthorizedKey([]byte(principal))
+		key, _, _, _, err := ssh.ParseAuthorizedKey([]byte(principal))
 		if err != nil {
 			fmt.Println(err)
 			os.Exit(1)
 		}
 
-		pkType := strings.ToUpper(strings.Split(pak.Type(), "-")[1])
-		fmt.Printf("Good \"%s\" signature for %s with %s key %s\n", namespace, principalEmail, pkType, ssh.FingerprintSHA256(pak))
+		keyType := strings.ToUpper(strings.Split(key.Type(), "-")[1])
+		fmt.Printf("Signature \"%s\" for \"%s\" with %s key %s is good.\n", namespace, principalEmail, keyType, ssh.FingerprintSHA256(key))
```

Comment:
This should be reverted back, IMO. While the change is undoubtably better grammar, the original is the same output as what `ssh-keygen` produces. Given this binary is a replacement for it in the signing workflow, I feel the outputs should match what the user is expecting / used to. 

You can see the `ssh-keygen` version here: https://github.com/openssh/openssh-portable/blob/80a2f64b8c1d27383cc83d182b73920d1e6a91f1/ssh-keygen.c#L2850

Reply:
In addition `git` itself has error checking on the first word being 'Good' - https://github.com/git/git/blob/aab89be2eb6ca51eefeb8c8066f673f447058856/gpg-interface.c#L583